ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -174,6 +154,25 @@ class TabBarItemView: UIControl, TokenizedControlInternal {
         updateLayout()
     }
 
+    private struct Constants {
```

Comment:
I think this isn't quite what we had in mind. I think that rather than having all of these constants, we still want these values to come from tokens. But they should be included in the parent's tokens, then set on the child control by that parent.

Reply:
Better than my interpretation by a long shot!

Reply:
Yes. My understanding of the decision we made offline aligns with what @huwilkes mentioned:

We won't have the tokenization infrastructure in each of the TabBarItemView (such as the defaultTokens and overrideTokens). But the tokens for the items will still be defined in the TabBarViewTokens class.